src/content/docs/en/guides/images.mdx

```diff
@@ -657,7 +657,27 @@ If you were using the image integration in Astro v2.x, complete the following st
     })
     ```
 
-2. Migrate any existing `<Image />` components.
+2. Update types (if required).
+
+  If you had special types configured for `@astrojs/image` in `src/env.d.ts`, you may need to change them back to the default Astro types if your upgrade to v3 did not complete this step for you.
+
+  ```ts title="drc/env.d.ts" del={1} ins={2}
+  /// <reference types="@astrojs/image/client" />
+  /// <reference types="astro/client" />
+  ```
+
+  Similarly, update `tsconfig.json` if necessary:
+
+  ```json title="tsconfig.json" del={3} ins={4}
+  {
+    "compilerOptions": {
+      "types": ["@astrojs/image/client"]
+      "types": ["astro/client"]
+    }
+  }
+  ```
```

Comment:
``````suggestion
		If you had special types configured for `@astrojs/image` in `src/env.d.ts`, you may need to change them back to the default Astro types if your upgrade to v3 did not complete this step for you.

		```ts title="drc/env.d.ts" del={1} ins={2}
		 /// <reference types="@astrojs/image/client" />
		 /// <reference types="astro/client" />
		```

		Similarly, update `tsconfig.json` if necessary:

		```json title="tsconfig.json" del={3} ins={4}
		{
			"compilerOptions": {
			  "types": ["@astrojs/image/client"]
			  "types": ["astro/client"]
			}
		}
		```
``````